projects/dsp-ui/src/lib/viewer/values/text-value/text-value-as-xml/text-value-as-xml.component.spec.ts

```diff
@@ -0,0 +1,418 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { TextValueAsXMLComponent } from './text-value-as-xml.component';
```

Comment:
organize imports

Reply:
done in https://github.com/dasch-swiss/dsp-ui-lib/pull/134/commits/2e7b3456557533e626891e138655faa23a444808